### PR TITLE
Use pascal casing for partials

### LINKED ISSUE #127: Make is easier to steal the site and run on Linux
I setup a Statiq site based on the Chocolatey theme which worked fine on my Windows machine, but it failed to apply the theme when building on a Linux build agent. To work on Linux `_viewstart.cshtml` needed to be renamed to `_ViewStart.cshtml`

### PR DESCRIPTION
**Pull request by pascalberger**
Under Linux _ViewStart.cshtml needs to be pascal cased. To be consistent across all partials I changed casing for all.

Fixes #127

### CODE REVIEW

File: input/_Layout.cshtml [hunk 1]
```diff
@@ -135,7 +135,7 @@
                                 <nav class="navbar navbar-expand-md p-0">
                                     <div id="leftSidebar" class="collapse navbar-collapse">
                                         <div class="d-sm-none">
-                                            @Html.Partial("global-partials/_globalnavigation")
+                                            @Html.Partial("global-partials/_GlobalNavigation")
```
Comment (designxsteph) [inline input/_Layout.cshtml:138]: This `_globalnavigation` is being pulled in from [choco-theme](https://github.com/chocolatey/choco-theme) and is not housed in this repository, but is being pulled in with Gulp. Would referencing it with Pascal Case here cause issues if the actual file is still all lower case?

Comment (pascalberger) [reply input/_Layout.cshtml]: Sorry, missed that. In this case I think we should keep it lower-cased. I updated the PR.

**Comment by designxsteph** [comment]
No problem! I'm doing some work in choco-theme and will be putting in a PR soon, so I will get this file converted over to Pascal Case as well so everything is consistent.

File: input/_Layout.cshtml (hunk 1 above)
Comment (designxsteph) [reply input/_Layout.cshtml]: No problem! I'm doing some work in choco-theme and will be putting in a PR soon, so I will get this file converted over to Pascal Case as well so everything is consistent

**Review by gep13 — APPROVED** [review]
LGTM!

**Comment by gep13** [comment]
@pascalberger I have made a slight change to include the issue number in the commit message, but otherwise, this looks ready to be merged.

Thanks for taking the time to fix this!

**Comment by gep13** [comment]
@pascalberger

**Pull request merged**